Formula/b/behaviortree.cpp.rb

```diff
@@ -0,0 +1,25 @@
+class BehaviortreeCpp < Formula
+  desc "Behavior Trees Library in C++"
+  homepage "https://www.behaviortree.dev/"
+  url "https://github.com/BehaviorTree/BehaviorTree.CPP/archive/refs/tags/4.6.2.tar.gz"
+  sha256 "b0e7e53b27feae894e2df3f3faadfdd49f2108ebccfb1bd7cc0d405ffc56cecd"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "cppzmq"
+
+  uses_from_macos "sqlite"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DBTCPP_UNIT_TESTS=OFF", "-DBTCPP_EXAMPLES=OFF"
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build",
                    "-DCMAKE_INSTALL_RPATH=#{rpath}",
                    "-DBTCPP_UNIT_TESTS=OFF",
                    "-DBTCPP_EXAMPLES=OFF",
                    *std_cmake_args
```
Pass `std_cmake_args` last to make sure we don't inadvertently override it. We also need to set `CMAKE_INSTALL_RPATH` so the installed binaries are able to find the linked libraries.